### PR TITLE
[Testing:Submission] Add Tests to SimpleStat and Team models

### PR DESCRIPTION
**Pull request by cjreed121**
### Please check if the PR fulfills these requirements:

* [x] Tests for the changes have been added/updated (if possible)
* [ ] Documentation has been updated/added if relevant

### What is the current behavior?
SimpleStat and Team models have no PHPUnit testing.
Partially solves #5495 

### What is the new behavior?
There are now PHPUnit tests for those files.

### CODE REVIEW

File: site/app/libraries/Core.php [hunk 1]
```diff
@@ -244,7 +244,7 @@ public function loadForum() {
             throw new \Exception("Need to load the config before we can create a forum instance.");
         }
 
-        $this->forum = new Forum($this);
+        //$this->forum = new Forum($this);
```
Comment (shailpatels) [inline site/app/libraries/Core.php:247]: Should this be commented out? There are other mentions of a forum member in core which all seems unused, but I think that should be removed in its own PR instead of here.

**Review by shailpatels — CHANGES_REQUESTED** [review]

File: site/app/libraries/Core.php (hunk 1 above)
Comment (cjreed121) [reply site/app/libraries/Core.php:247]: I'll remove from this PR and make a new PR for it. We discussed it should be turned off from being created in Core since it is never used. We are still keeping the code so a refactor can be done in the future.

**Review by shailpatels — APPROVED** [review]
Looks good
